components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -5384,4 +5387,180 @@ protected String getSearchQuery(String searchQuery) throws APIManagementExceptio
         }
         return criteria;
     }
+
+    /**
+     * Method to get the user specified mediation sequence.
+     *
+     * @param apiIdentifier : The identifier of the api.
+     * @param type : Mediation type. {in, out, fault}
+     * @param name : The name of the sequence that needed.
+     * @return : The content of the mediation sequence.
+     */
+    public String getSequenceFile(APIIdentifier apiIdentifier, String type, String name) throws APIManagementException {
+
+        Resource requiredSequence;
+        InputStream sequenceStream;
+        String sequenceText = "";
+
+        try {
+            if (apiIdentifier != null && type != null && name != null) {
+                requiredSequence = getDefaultSequence(type, name);
+                if (log.isDebugEnabled()) {
+                    log.debug("Get the default sequences.");
```

Comment:
may be we can add more context to this log saying what is the sequence name?

Reply:
Added more verbose log in commit https://github.com/wso2/carbon-apimgt/pull/5114/commits/c6bc99c70b7ef3df0181e7565b77152ab729b066